packages/swayze/lib/src/widgets/headers/header_drag_and_drop_preview.dart

```diff
@@ -0,0 +1,349 @@
+import 'package:cached_value/cached_value.dart';
+import 'package:flutter/material.dart';
+import 'package:swayze_math/swayze_math.dart';
+
+import '../../core/style/style.dart';
+import '../../core/viewport_context/viewport_context_provider.dart';
+
+/// Renders the preview line and block of a header drag and drop action.
+class HeaderDragAndDropPreview extends StatelessWidget {
+  final Axis axis;
+  final SwayzeStyle swayzeStyle;
+  final Offset translateOffset;
+
+  const HeaderDragAndDropPreview({
+    Key? key,
+    required this.axis,
+    required this.swayzeStyle,
+    required this.translateOffset,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final lineColor = swayzeStyle.dragNDropPreviewLineColor;
+    final lineWidth = swayzeStyle.dragNDropPreviewLineWidth + 1;
```

Comment:
Why do we need to increment the width by one?

Reply:
We don't, it is left over code 😓